db/models/gamesModels.js

```diff
@@ -32,3 +32,18 @@ exports.selectReviewByID = (review_id) => {
             return review
         })
 }
+
+exports.selectCommentsByReviewID = (review_id) => {
+    return db
+        .query(`
+        SELECT * 
+        FROM comments 
+        JOIN reviews 
+        ON reviews.review_id = comments.review_id
+        WHERE reviews.review_id = $1`, [review_id]).then(({rows})=> {
+            if (rows.length === 0) {
```

Comment:
To pass the suggested test, you will need some more logic when you hit this condition. Have a think about what you will need to check before you reject the Promise.